.travis.yml

```diff
@@ -177,7 +182,6 @@ before_install:
 install:
   - source bin/install_travis.sh
 script:
-  - bin/travis_clang_format.sh
```

Comment:
Why was this removed? Such a change should be documented in the git commit message.

Reply:
This is a redundant line. It's already in line 82.